rest-api-spec/src/main/resources/rest-api-spec/test/cat.nodes/10_basic.yml

```diff
@@ -58,6 +58,19 @@
       $body: |
                /^      http \n ((\d{1,3}\.){3}\d{1,3}:\d{1,5}\n)+  $/
 
+  - do:
+      cat.nodes:
+          h: disk,diskAvail,diskTotal,diskUsed,diskUsedPercent
```

Comment:
why test both diskAvail and its alias? Maybe we could do `h: diskAvail,diskTotal,diskUsed,diskUsedPercent` here and have another test for the aliases: `h: disk,dt,du,dup` ?